test/database.ts

```diff
@@ -2525,6 +2561,35 @@ describe('Database', () => {
     it('should accept requestOptions', () => {
       const fakeCallback = sandbox.spy();
 
+      database.runPartitionedUpdate(
+        {
+          sql: QUERY.sql,
+          params: QUERY.params,
+          requestOptions: {priority: RequestOptions.Priority.PRIORITY_LOW},
+          directedReadOptions: fakeDirectedReadOptions,
```

Comment:
Why are we setting directedReadOptions  in runPartitionedUpdate?

Reply:
Removed